src/main/java/com/RegisterQuizApplication/Registration/LoginController.java

```diff
@@ -0,0 +1,23 @@
+package com.RegisterQuizApplication.Registration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin(origins = "*")
+public class LoginController {
+
+    @Autowired
+    UserRepository repository;
+    @GetMapping(path="login/{username}/{password}")
```

Comment:
This will be security issue. Never capture username/password in URL. It should be retrived from request Body. So, it cant be a GET api call.